### DISCUSSION TITLE
Using different width % in Box components within Flex component

### DISCUSSION THREAD

**Discussion by chjpark** [twilio-labs/paste#513] [General] [answered]
Is it possible to use widths other than 100% in the Box components? I have a Flex component with two Box components in it and I'd like for one of them to be about 60% of the screen and the other to be able 40% of the screen.

**Comment by SiTaggart** [comment, marked as answer]
Seems like Grid would be a good fit for you here. https://paste.twilio.design/layout/grid.

You could do something like a 7/5 column split using the span prop.
